payments.md

```diff
@@ -0,0 +1,117 @@
+## BusinessPayment
+
+## Payment collection [/BusinessPayments]
+Payments endpoint allows you to make a Payment anonymously or by logging in. Responses will be returned in JSON format only.
+
+### BusinessPayment
+Payment as a resource represents following information
+
+| Property | Type | As request | As response | Description |
+| :-------------------- | :---------- | :-------------------- | :-------------------- | ------------------------------------------------------------ |
+| id | String | `Mandatory` | `Returned always` | It is the Identifier which is required to track the payment for that particular account.|
+| cardId | String | `Manadatory` | `Returned always` | It is the Identifier for the card that is saved in memcache. |
+| cards | BusinessCards |`Optional` | `Returned always` | It is the BusinessCards endpoint which is cinvoked before payment resource to save the save or store in memcache to make the payment. [BusinessPayments/BusninessCards]. |
```

Comment:
As card identifier is already provided, no need to provide the card details again. You can get the card details based on the card id.